File: 2710-RemoveTrailingZerosFromaString/2710-RemoveTrailingZerosFromaString.cpp
// Last updated: 3/10/2026, 11:34:22 AM
1class Solution {
2public:
3    string removeTrailingZeros(string num) {
4        for(int i = num.size() - 1; i >= 0; i--) {
5            if(num[i] == '0') {
6                num = num.substr(0, i);
7            }
8            else {
9                break;
10            }
11        }
12        return num;
13    }
14};

File: 2710-RemoveTrailingZerosFromaString/2710-RemoveTrailingZerosFromaString.js
// Last updated: 3/10/2026, 11:32:17 AM
1/**
2 * @param {string} num
3 * @return {string}
4 */
5var removeTrailingZeros = function(num) {
6    for(let i=num.length-1;i>=0;i--){
7        if(num[i]=="0"){
8            num=num.slice(0,i);
9        }
10        else{
11            break;
12        }
13    }
14    return num
15};